pkg/downloader/downloader.go

```diff
@@ -291,14 +291,14 @@ func Cached(remote string, opts ...Opt) (*Result, error) {
 }
 
 // cacheDirectoryPath returns the cache subdirectory path.
-// - "url" file contains the url
-// - "data" file contains the data
+// - "url" file contains the url.
```

Comment:
I don't think comment lines that start with a `-` need to end with a `.`.

Can we exclude them from the linter? It seems to have an `exclude` options, so I would hope this would disable the rule for list (I haven't tested it):

```yaml
exclude:
- '^-'
```

Reply:
It can be excluded. However, I propose a better approach by adding three spaces `   ` before `-`. In that case, `pkgsite` (old `godoc`) recognizes lines starting `   -` as a list.

Without three spaces before `-` (see live [here](https://pkg.go.dev/github.com/lima-vm/lima@v0.22.0/pkg/downloader#WithExpectedDigest)):

<img width="975" alt="image" src="https://github.com/lima-vm/lima/assets/3228886/0d549ce1-0561-4657-9fed-827577a6c82e">

With spaces:

<img width="938" alt="image" src="https://github.com/lima-vm/lima/assets/3228886/6c9e6f2c-f461-4bf9-a2d4-e1e60b97973d">

Reply:
I didn't like adding the spaces when just looking at the code in the editor. But seeing how it affects `godoc` changed my mind.